nautobot/extras/models/change_logging.py

```diff
@@ -213,18 +213,32 @@ def get_snapshots(self):
 
         prior_change = self.get_prev_change(only=["object_data_v2", "object_data"])
 
-        if self.action != ObjectChangeActionChoices.ACTION_CREATE and prior_change is not None:
-            prechange = prior_change.object_data_v2
-            if prechange is None:
-                prechange = prior_change.object_data
+        # Populate the prechange field, create actions do not need to have a prechange field
```

Comment:
Can we skip the `get_prev_change()` call in the CREATE case to make things slightly more efficient?